### ISSUE TITLE
Possibility of obtaining higher-order tangents in a single forward AD call (jax.jvp) ?

### ISSUE THREAD

**Issue by YingHH1** [google/jax#18056]
Hi,

I understand that a single forward AD in the direction v on the loss f gives the gradient vector dot product g⋅v; if we apply it again to g⋅v, we get Hv⋅v. There are two questions now:

1) Is it possible to do this using only the jax.jvp API? If not possible then what is the best practice to implement this?
2) Is it possible to do this by only calling jax.jvp once? This is (correct me if I'm wrong) because in dual number system, it is possible to extend the tangents to higher order traces

Many thanks!

**Comment by YingHH1** [comment]
Perhaps I should clarify a little more, my ultimate goal is to obtain the hessian vector vector dot product Hv⋅v in a single forward mode with one function evaluation (NOT hessian vector product Hv which needs access to the gradient g with many function evaluations in forward mode). This should be possible to obtain in theory, but I am unsure if this is implemented in Jax.

**Comment by YingHH1** [comment]
Found the solution here: https://github.com/google/jax/issues/924

I thought it would be good if this is included in the autodiff documentation :)
